media/css/security/security.scss

```diff
@@ -2,16 +2,12 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-$font-path: '/media/protocol/fonts';
-$image-path: '/media/protocol/img';
-
-@import '~@mozilla-protocol/core/protocol/css/includes/lib';
```

Comment:
We don't need to import the lib here because we're not using any lib members (functions, variables, mixins) in this file. All the partials import lib because they are using lib members (usually variables)